packages/core/content-manager/server/src/services/permission-checker.ts

```diff
@@ -9,74 +10,81 @@ const ACTIONS: any = {
   unpublish: 'plugin::content-manager.explorer.publish',
 };
```

Comment:
typescript trick !
can you add `as const` at the end? that way the type will contain the list of actions and not be `{[key: string]: string}`